lib/src/cli.dart

```diff
@@ -0,0 +1,119 @@
+
+library grinder.src.cli;
+
+import 'dart:async';
+import 'dart:convert' show JSON, UTF8;
+import 'dart:io';
+
+import 'package:args/args.dart';
+
+import 'singleton.dart';
+import 'utils.dart';
+import '../grinder.dart';
+
+// This version must be updated in tandem with the pubspec version.
+const String APP_VERSION = '0.6.6+3';
```

Comment:
Hmm, nevermind - I see this was a library private variable in the main lib, and is now a public var in a private library. So, a bit of cleanup :)

Let's adjust https://github.com/google/grinder.dart/blob/master/pubspec.yaml#L2 to match.